src/boehm_gc.c

```diff
@@ -39,13 +40,25 @@
 
 Int SyStorKill;
 
+// fill in the data "behind" bags
+struct OpaqueBag {
+    void * body;
+};
+GAP_STATIC_ASSERT(sizeof(void *) == sizeof(struct OpaqueBag),
```

Comment:
Just to understand, this Assert checks if the Byte size of a pointer and the byte size of the struct is the same. How can it not be?

Reply:
Technically, structs in C can be bigger than just their members.

This is more common with multiple members, for example {int x; char c;} is size 8 on my computer, not 4+1=5 as you may expect.

I would be very suprised is there was a computer where this assert was false, but the C standard technically allows it.